Remotion/ObjectBinding/Core/BindableObject/MixinIntroducedPropertyInformation.cs

```diff
@@ -132,20 +132,22 @@ public void SetValue (object instance, object value, object[] indexParameters)
 
     public IMethodInformation GetGetMethod (bool nonPublic)
     {
-      return Maybe
-          .ForValue (_interfaceImplementationPropertyInfo.GetGetMethod (nonPublic))
-          .Select (mi => mi as InterfaceImplementationMethodInformation)
-          .Select (mi => new MixinIntroducedMethodInformation (mi))
-          .ValueOrDefault ();
+      var getMethod = _interfaceImplementationPropertyInfo.GetGetMethod (nonPublic) as InterfaceImplementationMethodInformation;

```

Comment:
Should we split those statements? 
var getMethod
var interfaceImplementionGetMethod